app/views/katello/subscription_mailer/subscription_expiry.text.erb

```diff
@@ -13,3 +13,5 @@
   <%= _("Days Remaining") %>:       <%= pool.days_until_expiration %>
 
 <% end %>
+
+<%= _("View a report of the affected hosts") %>:               <%= @report_url %>
```

Comment:
We should only render this line if we have a `@report_url`

Reply:
ah yes. will include an if condition there.